### DISCUSSION TITLE
posts/2023/08/xiao-gao/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#18] [General]
# posts/2023/08/xiao-gao/

小高的小指留着长长的指甲。

https://guozheng.netlify.app/posts/2023/08/xiao-gao/

**Comment by liechi** [comment]
> 感情开盲盒嘞。

敢情？

**Reply by residualsun1** [reply]
写错了∑( 口 ||，已更正。

---

另外，我申请到rbind.io域名啦，现在是https://guozheng.rbind.io

**Reply by DylanYu233** [reply]
好好好，知道了知道了知道了
